pkg/overlay/cache.go

```diff
@@ -48,7 +48,8 @@ type DB interface {
 	KnownUnreliableOrOffline(context.Context, *NodeCriteria, storj.NodeIDList) (storj.NodeIDList, error)
 	// Paginate will page through the database nodes
 	Paginate(ctx context.Context, offset int64, limit int) ([]*NodeDossier, bool, error)
-
+	// VetNode returns whether or not the node reaches reputable thresholds
+	VetNode(ctx context.Context, id storj.NodeID, criteria *NodeCriteria) (bool, error)
```

Comment:
This would probably benefit from a better name. It's only querying and not actually updating it, hence the active verb seems out of place. Maybe `IsVetted` (no need to specify "Node" because all of the nodes are about nodes)?